webpack/scenes/Hosts/ChangeContentSource/index.js

```diff
@@ -0,0 +1,4 @@
+import { withRouter } from 'react-router-dom';
+import ChangeContentSourcePage from './ChangeContentSourcePage';
+
+export default withRouter(ChangeContentSourcePage);
```

Comment:
Not sure where you're using the props that `withRouter` provides? But React Router has added [hooks](https://v5.reactrouter.com/web/api/Hooks) such as `useLocation` which should mean that you don't need to use `withRouter` anymore.